docs/streams/developer-guide/config-streams.html

```diff
@@ -600,8 +613,34 @@ <h4><a class="toc-backref" href="#id33">default.windowed.value.serde.inner</a><a
           <span id="streams-developer-guide-max-task-idle-ms"></span><h4><a class="toc-backref" href="#id28">max.task.idle.ms</a><a class="headerlink" href="#max-task-idle-ms" title="Permalink to this headline"></a></h4>
           <blockquote>
             <div>
-              The maximum amount of time a task will idle without processing data when waiting for all of its input partition buffers to contain records. This can help avoid potential out-of-order
-              processing when the task has multiple input streams, as in a join, for example. Setting this to a nonzero value may increase latency but will improve time synchronization.
+              <p>
+                In a task with multiple input partitions (as in a join or merge), this is the amount
+                of time to block processing to wait for new data to be produced to a partition we
```

Comment:
Maybe, "to a partition that is caught up"? (to remove "we" from the sentence)

Reply:
I see what you mean. I also just noticed I've stacked up four prepositional phrases (!!!). I'll re-work it.